pkgs/servers/gotify/default.nix

```diff
@@ -18,10 +18,10 @@ buildGoModule rec {
     owner = "gotify";
     repo = "server";
     rev = "v${version}";
-    sha256 = "0zrylyaxy1cks1wlzyf0di8in2braj4pfriyqa24vipwrlnhvgs6";
+    sha256 = "0pkws83ymmlxcdxadb1w6rmibw84vzhx9xrhxc6b1rjncb80l0kk";
```

Comment:
the version in this file should get bumped as well, I think you're running into a caching issue if this passes locally